src/cfn.py

```diff
@@ -233,7 +233,6 @@ def download_file(stackname, path, destination=None, allow_missing="False", use_
     """
     if not destination:
         destination = '.'
-    utils.mkdirp(destination)
```

Comment:
but non existing targets are a supported use case like in `scp host:/home/elife/file.txt file.txt`